app/controllers/saml_idp_controller.rb

```diff
@@ -1,3 +1,5 @@
+# TODO: add unit tests for this controller
+# :nocov:
 class SamlIdpController < SamlIdp::IdpController
```

Comment:
Adding the two spec files made this file being read by simplecov. Before that, it was just ignored